### ISSUE TITLE
Login function

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#27]
## User story
1. As a user
2. I want / need to be able to log into my account and get access to different features depending on my status (admin?)
3. So that I can use the shared desk system to book desks and more

## Acceptance criteria
* Login function gets email and password as input
* Login function should correctly identify registered users in the database and let them access the system if their login data was correct
* Login function should return if the login was sucessful or not

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
Later issue: Login function should prevent simple attacks, such as outsiders entering the system due to unencrypted communication with the authentification server
